.github/workflows/release_nightly.yml

```diff
@@ -189,6 +189,16 @@ jobs:
         run: |
           lipo -create -output package/ruffle target/x86_64-apple-darwin/release/ruffle_desktop target/aarch64-apple-darwin/release/ruffle_desktop
           chmod +x package/ruffle
+      
+      - name: Create app bundle
+        run: |
+          cp -r desktop/packages/macOS package/Ruffle.app
+          mkdir package/Ruffle.app/Contents/MacOS
+          cp package/ruffle package/Ruffle.app/Contents/MacOS/ruffle
```

Comment:
Not critical, but we can avoid this `cp` by creating the `ruffle` executable in `package/Ruffle.app/Contents/MacOS/` in the first place. See the `Make universal binary` step above.

Reply:
I agree, but I didn't want to do this until we had this merged and confirmed that the packaged app bundle was being built correctly.